doc/en/developer/source/policies/pull_request.rst

```diff
@@ -0,0 +1,106 @@
+.. _pull_request:
+
+Pull Requests
+=============
+
+As with most open source project GeoServer is very happy to accept pull requests from the community. Contributions from the community have been a source of some of the best bug fixes and improvements in GeoServer and is a great way to give back to the project.
+
+This document is a guide designed to help contributors in successfully submitting a pull request:
+
+.. note:: 
+
+   We ask all fixes applied to master first, and then back ported to the stable and maintenance branches. 
+   
+   We require the following for fixes submitted via pull requests:
+
+   1. Required: Prepare your fix for the master branch
+   2. Required: Check the following (and comment in your pull request, or jira issue):
+      
+      * Investigate if the issue impact all the active branches (master / stable / maintenance)
+      * Investigate the your fix can be backported to all the active branches (master / stable / maintenance)
```

Comment:
* Investigate whether the fix can be backported to the applicable active branches (master / stable / maintenance)